src/stdune/path.ml

```diff
@@ -686,8 +676,8 @@ module Build = struct
         ]
 
   let is_alias_stamp_file s =
-    String.is_prefix (Local.to_string s) ~prefix:".aliases/"
-
+    String.is_prefix (Local.to_string s) ~prefix:".aliases/"  
```

Comment:
Unrelated, but I think we should just move this function the alias module.

Reply:
Added a CR-someday comment.